app/src/main/java/com/example/android_template/example/TeamRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.android_template.example
+
+import kotlinx.coroutines.delay
+
+interface ITeamRepository {
+    suspend fun getMyTeams(): List<String>
+}
+
+class TeamRepository : ITeamRepository {
+    override suspend fun getMyTeams(): List<String> {
+        delay(1000)
+        return listOf(
```

Comment:
```suggestion
       // TODO remove later
        delay(1000)
        return listOf(
```
Add TODO để sau này nhớ mà xoá